src/main/resources/view/PersonListCard.fxml

```diff
@@ -28,7 +28,7 @@
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
       </HBox>
       <FlowPane fx:id="tags" />
-      <Label fx:id="roles" styleClass="cell_small_label" text="\$roles" />
+      <!--      <Label fx:id="roles" styleClass="cell_small_label" text="\$roles" />-->
```

Comment:
i agree that removing the label makes the ui better as the tag fulfils the same role!